app/src/main/java/org/xwiki/android/sync/activities/BaseViewFlipper.java

```diff
@@ -29,22 +30,44 @@
  * BaseViewFlipper
  */
 public abstract class BaseViewFlipper {
+    /**
+     * Activity for operations
+     */
     protected AuthenticatorActivity mActivity;
+    /**
+     * Context of operations executing. In fact, represent {@link #mActivity}
+     */
     protected Context mContext;
+    /**
+     * Root view of flipper
+     */
     private View mContentRootView;
 
-    public BaseViewFlipper(AuthenticatorActivity activity, View contentRootView) {
+    public BaseViewFlipper(
```

Comment:
Would make more sense to document the parameter here for whoever is going to use this class than the private fields where its ending up (of course both is even better :)).